spec/factories/sp_upgraded_facial_match_profiles.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
-  factory :sp_upgraded_biometric_profile do
+  factory :sp_upgraded_facial_match_profile do
```

Comment:
we use this in `spec/jobs/reports/idv_legacy_conversion_supplement_report_spec.rb` so we need to rename there too right?

Reply:
oh gosh sorry i missed that!

Reply:
updated in https://github.com/18F/identity-idp/pull/11301/commits/e02200886a7d38060dc023f47ffad2afe4e31a78